code/app/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("com.diffplug.spotless")
+}
+
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    kotlin {
+        target("src/*/java/**/*.kt")
+        ktlint("0.41.0")
+        licenseHeaderFile("../../config/formatter/adobe.header.txt")
+    }
+}
+
+android {
+    namespace = "com.adobe.marketing.mobile.edge.bridge.testapp"
+
+    defaultConfig {
+        applicationId = "com.adobe.marketing.mobile.edge.bridge.testapp"
+        minSdk = BuildConstants.Versions.MIN_SDK_VERSION
+        compileSdk = BuildConstants.Versions.COMPILE_SDK_VERSION
+        targetSdk = BuildConstants.Versions.TARGET_SDK_VERSION
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        getByName(BuildConstants.BuildTypes.RELEASE) {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = BuildConstants.Versions.JAVA_SOURCE_COMPATIBILITY
+        targetCompatibility = BuildConstants.Versions.JAVA_TARGET_COMPATIBILITY
+    }
+
+    kotlinOptions {
+        jvmTarget = BuildConstants.Versions.KOTLIN_JVM_TARGET
+        languageVersion = BuildConstants.Versions.KOTLIN_LANGUAGE_VERSION
+        apiVersion = BuildConstants.Versions.KOTLIN_API_VERSION
+    }
+}
+
+dependencies {
+    implementation(project(":edgebridge"))
+    // TODO: Use 3.x versions for testing
+    implementation("com.adobe.marketing.mobile:core:2.+")
+    implementation("com.adobe.marketing.mobile:assurance:2.+")
+    implementation("com.adobe.marketing.mobile:edge:2.+")
+    implementation("com.adobe.marketing.mobile:edgeidentity:2.+")
+
+    implementation(BuildConstants.Dependencies.ANDROIDX_CORE_KTX)
```

Comment:
For test apps, let us not reference dependencies from BuildConstants.  This will let us switch/update dependencies easily.